expression-language/src/test/java/io/zeebe/el/EvaluationResultTest.java

```diff
@@ -102,6 +104,23 @@ public void periodExpression() {
     assertThat(evaluationResult.getList()).isNull();
   }
 
+  @Test
+  public void dateTimeExpression() {
+    final var evaluationResult = evaluateExpression(
+        "=date and time(\"2020-04-01T10:31:10@Europe/Berlin\")");
```

Comment:
Please add another test case to verify that a local date-time is transformed into a zoned date-time.